weaknesses/MASVS-PLATFORM/MASWE-0067.md

```diff
@@ -15,7 +15,23 @@ draft:
   description: not setting android:debuggable="false" on Android or get-task-allow="true"
     in the entitlements file on iOS
   topics: null
-status: draft
+status: new
 
 ---
 
+## Overview
+
+The `android:debuggable` flag in the `AndroidManifest.xml` file of an Android app, along with the `get-task-allow` attribute in the iOS app's `entitlements.plist`, indicates whether the application can be debugged. If these flags remain enabled in the app's production build, it opens the door for attackers to attach debuggers, examine the app's memory, alter code during execution, and effectively reverse engineer the application.
```

Comment:
```suggestion
Mobile apps typically include a configuration flag that determines whether the app is debuggable. While this flag is essential during development, leaving it enabled in production can introduce serious security risks. A debuggable app allows attackers to attach debuggers, inspect memory, manipulate execution flow, and bypass client-side controls, even on non-rooted or non-jailbroken devices. It may also expose sensitive information through verbose logging or developer tools that would otherwise be inaccessible.
```